localisation/00_wc_vanilla_replacements.csv

```diff
@@ -340,3 +340,11 @@ EVTOPTCHFP21041;I will tell [From.GetHerHim] of my prosperous realm.;Je vais lui
+REL_LOVER_MALE;lover;amant;Geliebter;;amante;;;;;;;;;x
+REL_LOVER_FEMALE;lover;amante;Geliebte;;amante;;;;;;;;;x
+depopulated_1_desc;This county has been depopulated to a minor extent. As times improve it will repopulate quickly.;La population de ce comt� a chut� suite � une �pid�mie. Quand les maladies ne le frapperont plus, celui-ci se repeuplera rapidement.;Die Bev�lkerungsanzahl in dieser Provinz ist leicht zur�ckgegangen. Wenn sich die Ziten bessern, wird sich die Bev�lkerung schnell erholen.;;Esta provincia ha sido despoblada a escala menor a causea de una epidemia. Cuando la enfermedad deje la provincia, se repoblar� r�pidamente.;;;;;;;;;x
+depopulated_2_desc;This county has been significantly depopulated. When times improve, it will repopulate over several years.;La population de ce comt� a chut� mod�r�ment suite � une �pid�mie. Quand les maladies ne le frapperont plus, celui-ci se repeuplera pendant plusieurs ann�es.;Die Bev�lkerungsanzahl in dieser Provinz ist erheblich zur�ckgegangen. Wenn sich die Zeiten bessern, wird sich die Bev�lkerung im Lauf einiger Jahre erholen.;;Esta provincia ha sido significativamente despoblada a causa de una epidemia. Cuando la enfermedad deje la provincia, se repoblar� al cabo de unos a�os.;;;;;;;;;x
+depopulated_3_desc;This county has been depopulated to a severe extent. When times improve, it will take many years to repopulate.;La population de ce comt� a chut� fortement suite � une �pid�mie. Quand les maladies ne le frapperont plus, celui-ci se repeuplera pendant de nombreuses ann�es.;Die Bev�lkerungsanzahl in dieser Provinz ist extrem zur�ckgegangen. Wenn sich die Zeiten bessern, wird sich die Bev�lkerung nur �ber viele Jahre hinweg erholen.;;Esta provincia se ha despoblado muy gravemente a causa de una epidemia. Cuando la enfermedad deje la provincia, llevar� muchos a�os hasta repoblarse.;;;;;;;;;x
+wargame_timer;Holding War Games;;Veranstaltet Kriegsspiele;;;;;;;;;;;x
+emperor_title_queen_mother_female;High Queen Dowager;;Hochk�niginwitwe;;;;;;;;;;;x
+focus_hunting_desc;[This.Religion.GetHighGodName] has populated the world with wondrous creatures. [This.GetTitledFirstName] travels the realm, making friends, and enjoying the hunt of these creatures.;[This.Get_Le_TitledFirstName] voyage dans le pays, tout en se faisant des amis et en tuant des cr�atures con�ues par [This.Get_le_HighGod].;[This.Religion.GetHighGodName] hat die Welt mit wundersamen Gesch�pfen bev�lkert. [This.GetTitledFirstName] bereist das Reich, gewinnt Freunde und findet Gefallen an der Jagd dieser Gesch�pfe.;;[This.GetTitledFirstName] viaja por el mundo haciendo amigos y matando criaturas de [This.Religion.GetRandomGodName];;;;;;;;;x
```

Comment:
It would be better if God’s name was not mentioned here at all. You may worship Death or something worse.

Reply:
Hmm, this was taken straight from vanilla, so there it was god/odin/allah etc. and that was fine... Should we replace it with titans/other creators? Or get rid of gods completely and speak of nature being filled with wondrous creatures?

Reply:
I would mention that you hunt `Azeroth's children`.

Reply:
Sounds good, will do